source/plugins/go/src/oms.go

```diff
@@ -1124,6 +1126,20 @@ func PostDataHelper(tailPluginRecords []map[interface{}]interface{}) int {
 	}
 	DataUpdateMutex.Unlock()
 
+	if IsAADMSIAuthMode == true && !IsGenevaLogsIntegrationEnabled {
+		if MdsdMsgpUnixSocketClient == nil {
+			Log("Error::mdsd::mdsd connection does not exist. re-connecting ...")
+			CreateMDSDClient(ContainerLogV2, ContainerType)
+			if MdsdMsgpUnixSocketClient == nil {
+				Log("Error::mdsd::Unable to create mdsd client. Please check error log.")
+				ContainerLogTelemetryMutex.Lock()
+				defer ContainerLogTelemetryMutex.Unlock()
+				ContainerLogsMDSDClientCreateErrors += 1
+				return output.FLB_RETRY
+			}
+			ContainerLogSchemaV2 = extension.GetInstance(FLBLogger, ContainerType).IsContainerLogV2()
```

Comment:
You would need to implement for IsContainerLogV2 for Windows also. you would need to update ingestion_token_utils.go for windows

Reply:
do we want to add it here or cover it when dealing with windows?